deploy/playbooks/examples/deploy_test.yml

```diff
@@ -18,18 +18,21 @@
      development_server: false
      purge_repos: True
      fqdn: "{{ inventory_hostname }}"
+     ## required. use something like chacra# so rabbitmq will start
```

Comment:
these changed lines aren't needed, mind rebasing this out of the PR?

Reply:
what? why?

I made comments double octothorped and commented variables one octothorped for clarity.

Reply:
I can't read. That is why
![hxihgck](https://user-images.githubusercontent.com/317847/32341717-a6f0af4a-bfd4-11e7-8934-0c3230da990a.gif)